packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -501,6 +501,10 @@ function emitBodyParameter(
       isBinaryPayload: isBinaryPayload(context, body.type, contentTypes),
       optional: !isBodyRequired(httpOperation.parameters)
     };
+  } else if (body.bodyKind === "multipart") {
```

Comment:
I am not sure I follow the fixes here? we already filtered out the bodyKind for single type which is what we can handle in line 485.

Reply:
This is more for comment purpose. because this is where to handle mfd.